content/en/opentelemetry/ingestion_sampling_with_opentelemetry.md

```diff
@@ -17,83 +17,101 @@ further_reading:
 
-## Reducing your ingestion volume
+Choose this method if you prefer a simpler setup without the need for a separate OpenTelemetry Collector service. To configure the Datadog Agent to receive traces using OTLP, follow the instructions on [OTLP Ingestion by the Datadog Agent][15].
+
+## Reducing ingestion volume
 
 With OpenTelemetry, you can configure sampling both in the OpenTelemetry libraries and in the OpenTelemetry collector: 
-- **Head-based Sampling** in the OpenTelemetry SDKs
-- **Tail-based Sampling** in the OpenTelemetry Collector
 
-{{< img src="/opentelemetry/guide/ingestion_otel/otel_head_tail_based_sampling.png" alt="Otel APM Metrics computation" style="width:100%;" >}}
+- **Head-based sampling** in the OpenTelemetry SDKs
+- **Tail-based sampling** in the OpenTelemetry Collector
+- **Probabilistic sampling** in the Datadog Agent
+
+### Head-based sampling
 
-### SDK-level sampling
+At the SDK level, you can implement _head-based sampling_. This is when the sampling decision is made at the beginning of the trace. This type of sampling is particularly useful for high-throughput applications, when you have a clear understanding of which traces are most important to ingest and want to make sampling decisions early in the tracing process.
 
-At the SDK level, you can implement _head-based sampling_, which is when the sampling decision is made at the beginning of the trace. This type of sampling is particularly useful for high throughput applications, for which you know that you do not need visibility over 100% of the traffic to monitor the application health. It can also help control the overhead introduced by OpenTelemetry.
+#### Configuring
 
-[TraceIdRatioBased][5] and [ParentBased][6] are the SDK's built-in samplers that allow you to implement deterministic head-based sampling based on the `trace_id` at the SDK level.
+To configure head-based sampling, use the [TraceIdRatioBased][5] or [ParentBased][6] samplers provided by the OpenTelemetry SDKs. These allow you to implement deterministic head-based sampling based on the `trace_id` at the SDK level.
 
-With head-based sampling, the APM metrics are computed **on the sampled traffic**, since only the sampled traffic is sent to the OpenTelemetry Collector or Datadog Agent, which is where the metrics calculation is done.
+#### Considerations
 
-To get accurate stats, you can upscale the metrics by using [formulas and functions][7] in Datadog dashboards and monitors, provided that you know the configured sampling rate in the SDK.
+Head-based sampling affects the computation of APM metrics because only the sampled traces are sent to the OpenTelemetry Collector or Datadog Agent, and this is where the metrics calculation is done.
+
+To get accurate metrics, upscale them using [formulas and functions][7] in Datadog dashboards and monitors, provided that you know the configured sampling rate in the SDK.
```

Comment:
```suggestion
To approximate real metrics from sampled metrics, use [formulas and functions][7] with the sampling rate configured in the SDK.
```
- phrasing it as "approximate" because that's sort of the best you can do with upscaling
- "real" may not be the best word here, I've just suggested it as a placeholder. is there a better way to describe "actual" (that is, not sampled) metrics? 
- I'm not sure about the use of the word "upscale" here. I've definitely seen it used at Datadog to talk about going from sampled data to real/actual data—but outside of DD, I usually see it in the context of AI models. Really not sure if customers will understand the meaning in the way we want them to

Reply:
Great points and suggestions. I think instead of real metrics, we could probably just say "unsampled metrics".

Reply:
oooh "unsampled metrics" is perfect!